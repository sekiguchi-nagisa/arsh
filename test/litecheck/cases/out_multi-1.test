#!/bin/sh
echo hello
echo world
echo !!
# CHECK: hello
# CHECK: worl
# CHECK: !!