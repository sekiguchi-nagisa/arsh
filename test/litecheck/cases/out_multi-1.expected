2c2
< worl
---
> world
stdout is mismatched
